components/AddLostItem.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import styled from "styled-components";
+import { useState } from "react";
+import { lostItems } from "./Db";
+
+export default function AddLostItem() {
+  const [lost, setLost] = useState(lostItems);
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    const formdata = new FormData(event.target);
+    const data = Object.fromEntries(formdata);
+    const newLostItem = {
+      ...data,
+      id: Math.random().toString(36).substring(2),
+    };
+    setLost((prevLost) => [newLostItem, ...prevLost]);
+  };
+
+  return (
+    <>
+      <StyledForm onSubmit={handleClick}>
+        <h1>Lost Something?</h1>
+        <label>
+          <h2>Tell us more</h2>
+          <label htmlFor="name">
+            <input name="name" type="text" placeholder="Name" required />
+          </label>
+          <br />
+          <br />
+          <StyledTextarea
+            type="text"
+            name="description"
+            placeholder="Description"
+            rows="15"
+            cols="35"
+            required
+          />
+        </label>
+        <br />
+        <label htmlFor="location">
+          <br />
+          <h3>Whats your location?</h3>
+          <input name="location" type="text" placeholder="Location" required />
+          <br />
+        </label>
+        <br />
+        <label htmlFor="picture">
+          <h4>Upload a Picture</h4>
+        </label>
+        <input
+          name="picture"
+          type="file"
+          id="picture"
+          accept="image/png, image/jpeg"
+          onChange={setLost.img}
+        />
+        <br />
+        <br />
+        <br />
+        <button type="submit">Submit</button>
+      </StyledForm>
+      <StyledNewItems>
+        {lost.map((setLost) => {
+          return (
```

Comment:
Hi Jannik, you named the variable setLost like the setter-function of your state. That was a bit confusing for me. Maybe you can find a better name ;)